web/src/beta/features/Editor/tabs/map/SidePanel/index.tsx

```diff
@@ -0,0 +1,67 @@
+import Loading from "@reearth/beta/components/Loading";
+import SidePanelSectionField from "@reearth/beta/components/SidePanelSectionField";
+import Text from "@reearth/beta/components/Text";
+import SidePanelCommon from "@reearth/beta/features/Editor/SidePanel";
+import { useSceneFetcher } from "@reearth/services/api";
+import { PropertySchemaGroup } from "@reearth/services/gql";
+import { useT } from "@reearth/services/i18n";
+import { useTheme, styled } from "@reearth/services/theme";
+
+type Props = {
+  sceneId: string;
+};
+
+type GroupSectionFieldProps = {
+  groups: PropertySchemaGroup[];
+};
+
+const GroupSectionField: React.FC<GroupSectionFieldProps> = ({ groups }) => {
```

Comment:
I'm assuming you were already going to do this since this is still a draft, but just in case definitely make this a separate file